### PR TITLE
Sprint4/issue#50, add some simple tests for home fragments and sort the quiz list

### PR DESCRIPTION
**Pull request by nbaehler**

### CODE REVIEW

**Comment by brunnerant** [comment]
If you want I can try to factorize the two similar methods by creating a super class for tests that need to connect to the database

**Comment by nbaehler** [comment]
> If you want I can try to factorize the two similar methods by creating a super class for tests that need to connect to the database, so that CodeClimate stops complaining.

On one hand we actually need to prepare the same environment so maybe it makes sense to have that refactor. On the other hand it may be a bit complicated.

**Comment by nbaehler** [comment]
> If you want I can try to factorize the two similar methods by creating a super class for tests that need to connect to the database, so that CodeClimate stops complaining.

I actually did what you proposed in #62, we had the same issue there. We‘ll see if it works out when merging the code.

**Comment by nbaehler** [comment]
> If you want I can try to factorize the two similar methods by creating a super class for tests that need to connect to the database, so that CodeClimate stops complaining.

Finally I added a super class here as well that does what you suggested.

File: app/src/main/java/ch/epfl/qedit/view/home/HomeQuizListFragment.java [hunk 1]
```diff
@@ -68,6 +70,7 @@ private void startQuizActivity(String quizID) {
 
         public CustomAdapter(Context context, ArrayList<Map.Entry<String, String>> entries) {
             this.entries = entries;
+            Collections.sort(this.entries, CustomComparator);
```
Comment (nbaehler) [inline app/src/main/java/ch/epfl/qedit/view/home/HomeQuizListFragment.java]: @cjordan7 as you handled this one your PR I should probably remove all changes to HomeQuizListFragment, do you agree?

Comment (cjordan7) [reply app/src/main/java/ch/epfl/qedit/view/home/HomeQuizListFragment.java]: Yes

Comment (nbaehler) [reply app/src/main/java/ch/epfl/qedit/view/home/HomeQuizListFragment.java]: I removed the chages

**Pull request merged**